src/libos/src/vm/process_vm.rs

```diff
@@ -212,42 +212,44 @@ impl<'a, 'b> ProcessVMBuilder<'a, 'b> {
     fn init_elf_memory(elf_range: &VMRange, elf_file: &ElfFile) -> Result<()> {
         // Destination buffer: ELF appeared in the process
         let elf_proc_buf = unsafe { elf_range.as_slice_mut() };
-        let mut empty_offset_vec: Vec<(usize, usize)> = Vec::with_capacity(3); // usally two loadable segments
+        // Source buffer: ELF stored in the ELF file
+        let elf_file_buf = elf_file.as_slice();
+
+        let mut base_load_address_offset = elf_file.base_load_address_offset() as usize;
```

Comment:
base_load_address_offset is not a mut variable

Reply:
Done. Thanks for your review.